### PR TITLE
base: helpers: Added jinja_filters arg to load_template

### PR DESCRIPTION
**Pull request by nickethier**
This is a bit more of a discussion with working code, but happy to have it merged if it checks out.

In the ios driver I want to implement the set_users template, but I would need to either compute each ssh key's fingerprint, or wrap the text since ios doesn't allow any lines longer than 254. Adding a filter to the base class seemed like the wrong solution, so I came up with this. This way the ios driver could override `base.load_template` and inject the filters specific to the driver.

### CODE REVIEW

File: napalm/base/helpers.py [hunk 1]
```diff
@@ -61,8 +61,9 @@ def load_template(cls, template_name, template_source=None, template_path=None,
             loader = jinja2.FileSystemLoader(search_path)
             environment = jinja2.Environment(loader=loader)
 
-            for filter_name, filter_function in CustomJinjaFilters.filters().items():
-                environment.filters[filter_name] = filter_function
+            for d in [CustomJinjaFilters.filters(), jinja_filters]:
```
Comment (bewing) [inline napalm/base/helpers.py]: Maybe `for filter_name, filter_function in itertools.chain(CustonJinjaFilters.filters().items(), jinja_filters.items()):` ?

Comment (nickethier) [reply napalm/base/helpers.py]: Ah I see you've uncovered my lack of python experience!

**Comment by nickethier** [comment]
Anything else you folks need from me to get this merged? @ktbyers @mirceaulinic 
I've about got a PR ready to make use of this if you'd like to see that first.

**Comment by dbarrosop** [comment]
LGTM as well :)

**Pull request merged**